### PR TITLE
hack around broken rvm ruby detection

### LINKED ISSUE #838: Unexpected Behavior with ENV in Gemfile
I recently got a fresh clone of errbit and put it on my dev machine and cd into the directory after which I was greeted with:

``` Unknown ruby interpreter version (do not know how to handle): ENV[GEMFILE_RUBY_VERSION]ifENV[GEMFILE_RUBY_VERSION].```

I use rvm and do not have that environment variable set.  When I switch the conditional in the Gemfile to be block based instead of inline, the warning / message seems to go away.

Thoughts?

### PR DESCRIPTION
**Pull request by stevecrozz**
Fixes #838

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://stevecrozz-gemfile-ruby-version.23f39035947abfbf29f9f7741cb9c271.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://stevecrozz-gemfile-ruby-version.23f39035947abfbf29f9f7741cb9c271.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 RAILS_VERSION = '~> 4.1.8'
 
-ruby ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
+send :ruby, ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
```
Comment (arthurnn) [inline Gemfile]: would `ruby(ENV['GEMFILE_RUBY_VERSION']) if ENV['GEMFILE_RUBY_VERSION']` 
(adding parenthesis) fix the problem?

Comment (stevecrozz) [inline Gemfile]: sadly, I don't think so:
```bash
echo "ruby(ENV['GEMFILE_RUBY_VERSION']) if ENV['GEMFILE_RUBY_VERSION']" | command tr -d '\r' | sed -n "s/[[:space:]]+rescue[[:space:]]+nil$//; /^\s*ruby[[:space:](]/ {s/^\s*ruby//; s/[[:space:]()'\"]//g; p;}"
ENV[GEMFILE_RUBY_VERSION]ifENV[GEMFILE_RUBY_VERSION]
```

It seems that any line beginning with 'ruby' triggers this rvm mechanism

Comment (stevecrozz) [inline Gemfile]: We could change it to:
```ruby
if ENV['GEMFILE_RUBY_VERSION']; ruby ENV['GEMFILE_RUBY_VERSION']; end
```
if you like that better. Putting it all on one line seems to defeat rvm's regex.

Comment (arthurnn) [inline Gemfile]: yeah... thats good !

**Pull request merged**